phoenicis-javafx/src/main/java/org/phoenicis/javafx/components/library/control/LibraryFeaturePanel.java

```diff
@@ -100,6 +102,11 @@
      */
     private final ObjectProperty<ShortcutDTO> selectedShortcut;
 
+    /**
+     * The currently selected details panel action
+     */
+    private final ObjectProperty<DetailsPanelAction> selectedDetailsPanelAction;
```

Comment:
What is the meaning of "currently selected action"? If e.g. the `CloseAction` happens, it is over as soon as the event has been executed. So how could it possibly be "currently selected"?

Reply:
Maybe the word `action` is not exactly fitting...
The idea is that the `action` defines the currently opened details panel.
Do you have a better fitting name?

Reply:
`OpenDetailsPanel`? Then it would be `None` instead of `CloseAction`.

Reply:
Sounds good to me. I will rename the classes accordingly

Reply:
Done

Reply:
Do you also have a suggestion for the package name?

Reply:
Maybe something like `panelState`?

Reply:
camel case for the package name?

Reply:
I think so, yes. Do we have something else in another place?

Reply:
I don't know. I just don't think it is standard to use camel case in package names. It looks weird to me

Reply:
I don't mind really. If it's more standard to use kebab case, feel free. Do you have a source (just interested)?

Reply:
Kebab case is not allowed, because `-` is an illegal symbol for Java package names.

Reply:
Ok then how should it be?

Reply:
I don't know actually. we could go with one of the following:
- `state`
- `panel.state`
- `detail.panel.state`
- `panelstate`
- `detailstate`

I have no source for arguing against camel case in package names I have just never seen it being used.

Reply:
I wouldn't use the dot because it normally indicates the directory structure. Choose one of the last two (whatever you prefer).

Reply:
Done. I went for `panelstates`